README.md

```diff
@@ -126,8 +140,8 @@ Supported options:
 
 | Name  | Type     | Description |
 | :---- | :------: | :--- |
-| url | string   | URL you want to share (you can share a base64 file url only in iOS & Android ) |
-| urls | Array[string]   | URL's you want to share, Only for iOS and Android (you can share a base64 file url only in iOS & Android ) |
+| url | string   | URL you want to share (only support base64 string in iOS & Android ). |
+| urls | Array[string]   | array of base64 string you want to share (only support iOS & Android). |
```

Comment:
It supports file URI as well, not only base64 strings. Also, sharing multiple base64 strings is not an efficient way to share files.

Reply:
Thanks, @kkbhav.
Guess we need another PR.